src/inmanta/moduletool.py

```diff
@@ -308,6 +309,9 @@ def bump_version(cls, change_type: ChangeType, version: Version, version_tag: st
         )
         parts[bump_index] += 1
 
+        while len(parts) > 3 and parts[-1] == 0:
+            parts.pop()
```

Comment:
This removes the fourth digit always if it's 0, in contrast to what I put in my snippet. I don't have a strong opinion on this, just a slight preference for keeping four digits if there already were 4 or more. The main thing is that if we take this approach, the comment up above is no longer accurate: "use 4th digit only if it already existed or ...".

So either we need to update the comment or we need to change this approach to do as the comment states.